lib/vagrant-openshift/helper/command_helper.rb

```diff
@@ -50,6 +51,12 @@ def do_execute(machine, command, options={})
           end
         }
 
+        if options[:metainfo]
```

Comment:
@stevekuznetsov Might it be better to just use set -x if you want to see all the commands run?  I guess I have just personally never loved the vagrant output.